src/frontend/src/pages/HomePage.js

```diff
@@ -56,6 +57,7 @@ export const HomePage = ()=>{
                     latestBlogs && <BlogList blogList={latestBlogs.slice(0,Math.min(3,latestBlogs.length))} />
                 }
             </div>
+            <Footer/>
```

Comment:
Add this in `App.js` not here, the footer should be added below the routes
https://github.com/ken1000minus7/Bloggy/blob/529e6922998239ac9f6f09bd6d8408b2fddb502b/src/frontend/src/App.js#L30-L42